examples/with-parcel/package.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "community",
-  "type": "module",
+  "name": "with-parcel",
```

Comment:
Doesn't support `exports`.

Reply:
See https://github.com/status-im/status-web/pull/307#discussion_r962933933 and https://github.com/parcel-bundler/parcel/issues/4155.